.env.example

```diff
@@ -62,3 +62,6 @@ AWS_BUCKET=
 AWS_USE_PATH_STYLE_ENDPOINT=false
```

Comment:
(This commit doesn't have to do with this line of code. I'm just making a "conversation" that can be "resolved".)

The commits on this Pull request are as follows:

1. e205338 Revert "Polishing tests and begining to implement subscription"
1. 4a080d2 Revert "all feature tests completed and commented on time to subscribe"
1. 014fa09 Revert "test TODO comment edited line 74"
1. a5bb5f2 added updates section to README.md
1. c1fc8ab users without premium can only create 3 notes
1. 4b0d203 Update .env.example
1. 1cd4f31 Update README.md
1. 350ffbc Update README.md
1. 668e64c snapshot update
1. a2059b9 added subscription functionality and new button
1. 7ef0248 Stripe initialization
1. 77cf260 test TODO comment edited line 74
1. 4093c18 all feature tests completed and commented on time to subscribe
1. 523439c Polishing tests and begining to implement subscription
1. 6cccd55 personal test file begun and many comments added

I'm pretty sure we only need 4-11. Can you remove the others?

Reply:
I just closed this and then used git rebase -i <commit hash> in order to get rid of the commits that overlap with assignment A's PR. It took me a little while to complete the rebase process since there were a few conflicts with the head branch after getting rid of the overlapping commits.